packages/grid/_modules_/grid/models/gridOptions.tsx

```diff
@@ -378,7 +378,7 @@ export const DEFAULT_GRID_OPTIONS: GridOptions = {
   disableColumnMenu: !EXPERIMENTAL_ENABLED,
   disableColumnFilter: !EXPERIMENTAL_ENABLED,
   disableColumnSelector: !EXPERIMENTAL_ENABLED,
-  hideToolbar: !EXPERIMENTAL_ENABLED,
+  showToolbar: EXPERIMENTAL_ENABLED,
```

Comment:
We are documenting it, it's not longer experimental. 

```suggestion
  showToolbar: false,
```

Reply:
done